npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -496,6 +509,12 @@ def skip_package_lock?
         npmrc.content.match?(/^package-lock\s*=\s*false/)
       end
 
+      def skip_pnpm_lock?
+        return false unless npmrc
+
+        npmrc.content.match?(/^lockfile\s*=\s*false/)
```

Comment:
Could this `lockfile` key have whitespace in front of it and still work?

Reply:
I don't know, I followed the pattern of `skip_package_lock?`. This is an INI file, and I'm not sure if that's valid.

That said, this feature is questionable, since it'd be only useful if someone is committing a lockfile and configuring it to not be used. Maybe overkill to add a special case for this ourselves, so we could alternatively remove `skip_package_lock?` and not add this.

Reply:
I'm also cool with shipping as-is, and we can always later on add extra handling for whitespace if that becomes a problem. My guess is it never will.